src/components/suggest/_suggest_item.scss

```diff
@@ -62,7 +62,7 @@
 
   .ouiSuggestItem__label {
     @include ouiTextTruncate;
-    font-family: $ouiCodeFontFamily;
```

Comment:
Do we want to make `$ouiCodeFontFamily` point to `var(--oui-code-font-family)`? For backwards compatibility?

Reply:
I have that in a future commit. Lemme reorder the commits and update this.